SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1143,6 +1264,12 @@ void USpatialActorChannel::RemoveRepNotifiesWithUnresolvedObjs(TArray<UProperty*
 				continue;
 			}
 
+			// Only when there are unresolved refs.
+			if (ObjRef.Value.UnresolvedRefs.Num() == 0)
```

Comment:
Why is this check necessary? Doesn't the RefMap contain only unresolved references?

Reply:
Now it contains both.